### DISCUSSION TITLE
Using STAC for GeoJSON geomerty type: "Point"

### DISCUSSION THREAD

**Discussion by ax0n-pr1me** [radiantearth/stac-spec#1166] [General STAC Q&A]
Greetings,

I am working with time series point data and I think it will benefit us to use STAC Spec. Has anyone done this before? Did you hit any potholes or find any gotchas?

My initial considerations are that some of the required fields are not directly relevant, but could be used. It actually makes sense for us to buffer the points for the bbox, but the links and assets are required by STAC but not really applicable for us... Sometimes these locations have imagery associated with them but not aerial or orthometric, just basic mobile phone pictures usually. I would consider putting the references to these in the properties dictionary instead of assets or links, but I am not sure if that makes more sense.... 

Anyway - just looking for some feedback if anyone has done this before. I can always adapt it to work for us without required fields but would love to contribute if point data inclusion was interesting enough. 

Thank you all!

**Comment by m-mohr** [comment]
If assets are generally applicable for you and you are describing vector data, isn't OGC API - Features a better match for you?

Generally, if the vector geometry is the **data**, then you should likely use OGC API - Features.
If the vector geometry is the **metadata**, then STAC is likely the right match.
